pkgs/applications/graphics/cq-editor/default.nix

```diff
@@ -1,10 +1,10 @@
 { lib
 , mkDerivationWith
-, python3Packages
```

Comment:
please make the change in `all-packages.nix`, `python3Packages = python37Packages;`.

Reply:
Done, thankyou @FRidh